cognite/neat/graph/loaders/rdf_to_dms.py

```diff
@@ -40,6 +43,117 @@ def rdf2nodes_and_edges(
     data_model = DataModel.from_rules(rules)
     pydantic_models = rules_to_pydantic_models(rules)
 
+    for class_ in rules.classes:
+        if f"{rules.space}:{class_}" in data_model.containers:
```

Comment:
Suggest doing an early exit to avoid the extra indent (makes it more readable in my opinion)
```suggestion
        if f"{rules.space}:{class_}" not in data_model.containers:
            continue
```

Reply:
nice